core/src/test/java/org/infinispan/lock/InfinispanLockTest.java

```diff
@@ -41,7 +41,7 @@ public void testTimeout() throws InterruptedException {
       final String lockOwner1 = "LO1";
       final String lockOwner2 = "LO2";
 
-      final InfinispanLock lock = new InfinispanLock(commonPool(), AbstractCacheTest.TIME_SERVICE);
+      final InfinispanLock lock = new InfinispanLock(commonPool(), commonPool(), AbstractCacheTest.TIME_SERVICE);
```

Comment:
Please use `AbstractInfinispanTest.testExecutor()` in the tests, with the common pool it's impossible to filter logs by test.

Reply:
Sure, I can change the existing reference to the updated one.